src/emulator/env.ts

```diff
@@ -14,6 +15,7 @@ export function setEnvVarsForEmulators(
   env: Record<string, string | undefined>,
   emulators: EmulatorInfo[],
 ): void {
+  maybeUseMonospacePortForwarding(emulators);
```

Comment:
let's not use internal codenames in public facing code, just "maybeUsePortForwarding"